docs/ldoc.css

```diff
@@ -143,12 +143,15 @@ pre {
 }
 
 table.index {
-/* border: 1px #00007f; */
+    /* border: 1px #00007f; */
 }
-table.index td { text-align: left; vertical-align: top; }
 
-#navigation ul
-{
+table.index td {
+    text-align: left;
+    vertical-align: top;
+}
+
+#navigation ul {
     font-size:1em;
     list-style-type: none;
```

Comment:
Can I edit the PR to remove that line or can you add a fixup for it?

Reply:
i think the one which you wanted is on line 266